app/sidekiq/revalidate_old_link_check_reports_worker.rb

```diff
@@ -0,0 +1,46 @@
+class RevalidateOldLinkCheckReportsWorker
+  include Rails.application.routes.url_helpers
+  include Sidekiq::Job
+  MAX_REPORTS_TO_CHECK = 60_000
+  FIND_EACH_BATCH_SIZE = 50
+
+  sidekiq_options queue: "link_checks", retry: 6
+  # Stop retrying jobs within a day as new jobs are enqueued each day,
+  # and also retry less times within a day as this isn't a critical
+  # task.
+  sidekiq_retry_in do |retry_count, _exception|
+    # First retry in 15 seconds, 6th (and last) retry at ~5 hours
+    ((0.5 * (retry_count + 1)**4) * 30)
```

Comment:
Nitpick alert; I'm not sure if these multipliers are ever liable to change, but having them as constants after the batch size constant might make this more readable

Reply:
I didn't give this much of a read tbh (this code is copied over from the previous worker). I agree with you about the readability, but I think the comment helps.